### PR TITLE
HeadModel - Final Part

### PR DESCRIPTION
**Pull request by Hicham2012**

### CODE REVIEW

**Comment by tkalejandro** [comment]
@Hicham2012  There is something wrong in this PR..  maybe try to run `npm run build` and check locally what is actually wrong.

File: src/modules/Experience/models/HeadModel.tsx [hunk 1]
```diff
@@ -13,21 +13,40 @@ interface HeadModelProps extends GroupProps {
 
 const HeadModel = ({ selectedColor, map, mousemove = true, ...props }: HeadModelProps) => {
   const [gltfScene, setGltfScene] = useState<Group<Object3DEventMap> | null>(null);
+  const [gltfScene1, setGltfScene1] = useState<Group<Object3DEventMap> | null>(null);
+
   const { scene } = useGLTF('/models/head-3.glb');
+  const { scene1 } = useGLTF('/models/cone.glb')
```
Comment (tkalejandro) [inline src/modules/Experience/models/HeadModel.tsx]: MAybe is this scene1 your bug... are you sure scene1 exist inside this GLTF??

If you need many scenes maybe, dont destructure , you can try something like.  coneModel.scene  or as you wish.

Comment (Hicham2012) [reply src/modules/Experience/models/HeadModel.tsx]: I fixed the bug, thanks :)

File: src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ const Message = () => {
           )}
         </Box>
       </Flex>
-      <Clouds material={THREE.MeshBasicMaterial}>
+      {/* <Clouds material={THREE.MeshBasicMaterial}>
```
Comment (tkalejandro) [inline src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx:96]: So what is happening with this clouds??  

Are they in or out?? XD 


Over all did you test your solution mobile tablet and desktop?

Comment (Hicham2012) [reply src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx:96]: The clouds make the website break for me.
I made sure that solution is working on mobile and other devices in the next commit.

Comment (tkalejandro) [reply src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx:96]: So there are no clouds anymore ???

**Review by tkalejandro — APPROVED** [review]

File: src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx (hunk 2 above)
Comment (Hicham2012) [reply src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx:96]: We will keep them, but will optimize them in a separate ticket.

**Pull request merged**